app/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -59,7 +59,12 @@ const Summary = ({ experiment, refetch }: SummaryProps) => {
       changelogMessage: CHANGELOG_MESSAGES.REQUESTED_REVIEW_END_ENROLLMENT,
     },
     {
-      publishStatus: NimbusExperimentPublishStatusEnum.IDLE,
+      publishStatus:
+        experiment.isRollout &&
+        experiment.status === NimbusExperimentStatusEnum.LIVE &&
+        experiment.publishStatus === NimbusExperimentPublishStatusEnum.DIRTY
+          ? NimbusExperimentPublishStatusEnum.DIRTY
+          : NimbusExperimentPublishStatusEnum.IDLE,
```

Comment:
Oh is this to set what to write publishStatus to on the update flow?  Yeah exactly, now this logic is living in multiple places on the frontend.  No need for that.